### PR TITLE
recommedation files

### PR DESCRIPTION
**Pull request by AndreyPetlyashenko**

### CODE REVIEW

File: sections/findify-recommendation.liquid [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        prevEl: '.swiper-button-prev',
+      },
+    
+    });
+
+</script>
+
+
+
+{% comment %}
```
Comment (wolff95) [inline sections/findify-recommendation.liquid:79]: You can delete this comment

File: snippets/findify-product-card.liquid [hunk 2]
```diff
@@ -29,7 +29,7 @@
         height="{{ card_product.featured_media.height }}"
       />
     </div>
-    <div>{{ card_product.title }}</div>
+    <div class='findify-swiper-product-title'>{{ card_product.title }}</div>
```
Comment (wolff95) [inline snippets/findify-product-card.liquid:32]: This snippet is in common between autocomplete, search and recommendation.

Put a findify-product-title class

Then within your findify-recommendation use something in this lines:

`.recommendation-wrapper .findify-product-title {}`

File: assets/findify-recommendation.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+.swiper-container-vertical>.swiper-wrapper {
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+}
+
+.swiper-wrapperold {
```
Comment (wolff95) [inline assets/findify-recommendation.css:32]: Is this used?

If not, delete!

File: assets/findify-recommendation.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+.swiper-pagination-bullets.swiper-pagination-horizontal {
+    bottom: 0;
+    top: auto;
+    left: 0;
+    width: 100%;
+    z-index: 9;
+}
+
+/* grid */
+.findify-grid-container {
```
Comment (wolff95) [inline assets/findify-recommendation.css:139]: Is this used?

File: assets/findify-recommendation.css [hunk 5]
```diff
@@ -0,0 +1,159 @@
+.findify-element div[data-findify-product-card] {
+    cursor: pointer;
+}
+
+.findify-swiper {
+    display: none;
+}
+
+.findify-swiper.inited {
```
Comment (wolff95) [inline assets/findify-recommendation.css:9]: Is this used?

And call it .init or .initialized

Comment (wolff95) [reply assets/findify-recommendation.css:9]: Or .loaded 
or .ready

File: snippets/findify-product-card.liquid (hunk 2 above)
Comment (AndreyPetlyashenko) [reply snippets/findify-product-card.liquid:32]: ok)

**Review by wolff95 — APPROVED** [review]

**Pull request merged**